ivy/functional/frontends/__init__.py

```diff
@@ -3,6 +3,7 @@
 
 versions = {
     "torch": "2.0.1",
+    "torchvision": "0.15.2.",
     "tensorflow": "2.13.0",
     "numpy": "1.25.2",
     "jax": "0.4.14",
```

Comment:
Need to add `"test_torchvision"` to `submodules` and `"test_torchvision": ["torchvision", 27],` to `db_dict` in `run_tests_CLI/synchronize_db.py` and in `run_tests.py`.

Reply:
You forgot the changes in `run_tests.py`.